api/grpc/service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package pb;
+
+
+service WorkerService {
+  rpc Start        (JobRequest         ) returns (JobStatus       ) {}
+  rpc Status       (StatusRequest      ) returns (JobStatus       ) {}
+  rpc Stop         (StopRequest        ) returns (JobStatus       ) {}
+  rpc GetOuput     (GetOutputRequest   ) returns (stream JobOutput) {}
+}
+
+message StartRequest {
+  string id = 1;
```

Comment:
Hmm, is this a typo? Or what does "id" represent in the Start call?

Reply:
Seems like StartRequest is unused actually, can be removed?

Reply:
Thanks for pointing this out. It has been removed